fact-bounty-flask/api/app.py

```diff
@@ -9,7 +9,7 @@
 from flasgger import Swagger
 
 from api import commands
-from api import admin, user, stories, crawler, util
+from api import user, stories, crawler, util, admin
```

Comment:
I think the changes in this file are unnecessary and can be reverted to its original state to avoid drifting away from what this PR actually solves